Practice/kruglov/home_task_4/PyLec4T3-4.py

```diff
@@ -0,0 +1,34 @@
+# 3
```

Comment:
Good solution, but the task expects using of a single list.

Reply:
Noted how - "Poor solution"